Elaborate on validations made by the APIKit router (#71)
mulesoft/docs-apikit: modules/ROOT/pages/apikit-4-raml-flow-concept.adoc

== APIkit Router

> The APIkit router is a key message processor that routes messages from the incoming API request to the corresponding mule flows and serializes a response. It also enriches messages by adding default values to the messages. For example, APIkit returns a `Bad request` response if the request is invalid, and a `Not implemented` response if the API resource that you request is not associated with a flow.

Reviewer: ```suggestion
The APIkit router is a key message processor that routes messages from the incoming API request to the corresponding Mule flows and serializes a response. It also enriches messages by adding default values to the messages. For example, APIkit returns a `Bad request` response if the request is invalid, and a `Not implemented` response if the API resource that you request is not associated with a flow.
```

Reviewer 2: Good catch!
Thanks!

> Although the APIkit router can validate your request payload, headers, and query and URI parameters, note that the APIkit router does validate everything declared in an API specification. For example, the router does not enforce authentication because that should be provided by API Gateway policies that execute before your request reaches the router.

Reviewer: ```suggestion
Although the APIkit router can validate your request payload, headers, query, and URI parameters, the router does not validate everything declared in an API specification. For example, the router does not validate authentication because API Gateway policies, which execute before your request reaches the router, are responsible for enforcing authentication.
```

Reviewer: Feel free to suggest something else. I was trying to write around "because that should be provided by API Gateway policies" to avoid use of "should be". I also wondered whether "enforcement" is different than "validation".
